Source/Scene/CesiumIon.js

```diff
@@ -198,6 +203,70 @@ define([
                 }
 
                 return factory(endpoint.options);
+            })
+            .then(function(imageryProvider) {
+                return imageryProvider.readyPromise
+                    .then(function() { return imageryProvider; });
+            });
+    };
+
+    /**
+     * Asynchronously creates a {@link Cesium3DTileset} representing a Cesium ion 3D Tiles asset and
+     * waits for it to become ready.
+     *
+     * @param {Number} assetId The Cesium ion asset id.
+     * @param {Object} [options] An object with the following properties:
+     * @param {String} [options.accessToken=CesiumIon.defaultAccessToken] The access token to use.
+     * @param {String} [options.serverUrl=CesiumIon.defaultServerUrl] The url to the Cesium ion API server.
+     * @param {String} [options.tilesetOptions] Additional options to be passed to the {@link Cesium3DTileset} constructor.
+     * @returns {Promise<Cesium3DTileset>} A promise to the ready tileset representing the requested Cesium ion Asset.
+     *
+     * @example
+     * //Load a tileset with asset ID of 2347923
+     * Cesium.CesiumIon.create3DTileset(2347923)
+     *   .then(function (tileset) {
+     *     viewer.scene.primitives.add(tileset);
+     *   });
+     *
+     * //Load a tileset with asset ID of 2347923 for 3D Tile classification
+     * Cesium.CesiumIon.create3DTileset(2347923, { tilesetOptions: { classificationType: Cesium.ClassificationType.CESIUM_3D_TILE } })
+     *   .then(function (tileset) {
+     *     viewer.scene.primitives.add(tileset);
+     *   });
+     */
+    CesiumIon.create3DTileset = function(assetId, options) {
```

Comment:
Since you caved and added this helper function, I really think we should have helper functions for the remaining asset types too.

Reply:
Even if it seems silly, I think it will make the API easier to use.  It will be obvious which function to call to create your asset

Reply:
You may be right, but that's outside the scope of this PR.  If we do add support for them all, I would probably want to break them into individual function files rather than a utility class to avoid having to suck in a lot of code that's not being used.

Reply:
Okay.  I think it'd be good to do this before the release.